cmd/cmd.go

```diff
@@ -490,7 +499,7 @@ func generate(cmd *cobra.Command, model, prompt string, wordWrap bool, format st
 	return nil
 }
 
-func generateInteractive(cmd *cobra.Command, model string) error {
+func generateInteractive(cmd *cobra.Command, model string, wordWrap bool, format string) error {
```

Comment:
This probably deserves a struct instead of just passing in all of the variables

Reply:
Given `generate` takes them, would it be ok if we restructure in another PR? Also concerned that since we override these args in a loop in this function, it will look funny to change an options struct